charts/thoras/templates/api-server-v2/deployment.yaml

```diff
@@ -139,11 +139,11 @@ spec:
           - name: SERVICE_FORECAST_AFFINITY_THRESHOLD
             value: {{ .Values.thorasApiServerV2.forecastAffinityThreshold | default "2m" | quote }}
           - name: SERVICE_ENABLE_COST_REFRESH_BATCHING
-            value: {{ .Values.thorasApiServerV2.costRefreshBatching.enabled | quote }}
+            value: {{ .Values.costRefreshBatching.enabled | ternary "true" "false" | quote }}
```

Comment:
No need for ternary. Default is false via values.yaml. If the user wants to pass in true, they can. Will work either way with just `.Values.costRefreshBatching.enabled | quote`